lib/install/management/upgrade.go

```diff
@@ -233,7 +224,17 @@ func (d *Dispatcher) ensureRollbackReady(conf *config.VirtualContainerHostConfig
 		log.Infof("Roll back finished - Appliance is kept in powered off status")
 		return nil
 	}
-	return d.startAppliance(conf)
+	if err = d.startAppliance(conf); err != nil {
+		return err
+	}
+
+	ctx, cancel := context.WithTimeout(d.ctx, settings.Timeout)
+	defer cancel()
+	if err = d.CheckServiceReady(ctx, conf, nil); err != nil {
+		// do not return error in this case, to make sure clean up continues
+		log.Info("\tAPI may be slow to start - try to connect to API after a few minutes")
```

Comment:
the first place to check context deadline is to let user know the upgrade failure caused by timeout.
here, even we got timeout, user should just try to connect service few minutes later, cause this is for rollback only.
if rollback failed, we don't actually can do anything. And if the service works fine, rollback should as well.  I don't need to remind them retry with increased timeout.